app/keys/certificate.crt

```diff
@@ -0,0 +1,25 @@
+-----BEGIN CERTIFICATE-----
```

Comment:
Let's not commit the cert, since this will differ when we deploy different clusters.